src/ExEss/Cms/Component/Health/Handler/DoctrineHandler.php

```diff
@@ -16,10 +16,11 @@ public function __construct(Connection $connection)
 
     public function getHealthCheck(): HealthCheckResult
     {
-        if ($this->connection->ping() === false) {
```

Comment:
ah yes, I read somewhere ping was deprecated, no?

Reply:
indeed, so I already implemented the workaround so we won't get a problem later